### PR TITLE
Update hubspot plugin

### PR DESCRIPTION
**Pull request by bozhi**
- Update docstring
  - Change dataclass union type to any

### CODE REVIEW

File: hubspot/plugin.py [hunk 1]
```diff
@@ -608,6 +611,349 @@ def hubspot_update_contacts(
...
+military_status
+recent_conversion_event_name
+relationship_status
+school
+seniority
+start_date
+work_email
+firstname
+hs_analytics_first_url
+hs_email_optout_88607926
```
Comment (jngiam) [inline hubspot/plugin.py:715]: are these default across the platform?

```
hs_email_optout_88607926
hs_email_optout_96642417
```

File: hubspot/plugin.py [hunk 2]
```diff
@@ -1054,46 +1376,211 @@ def hubspot_update_companies(
...
+numberofemployees
+annualrevenue
+hs_parent_company_id
+hs_num_child_companies
+hubspotscore
+days_to_close
 
 Default Properties of Type Datetime:
-{", ".join(_COMPANY_PROPERTIES_DATETIME)}
+closedate_timestamp_earliest_value_a2a17e6e
```
Comment (jngiam) [inline hubspot/plugin.py:1503]: these values with a2a17e6e and other hashes feel like they might be generated for an account specifically

File: hubspot/plugin.py [hunk 3]
```diff
@@ -1054,46 +1376,211 @@ def hubspot_update_companies(
...
+hs_time_in_opportunity
+hs_time_in_other
+hs_time_in_salesqualifiedlead
+hs_time_in_subscriber
+hs_total_deal_value
+hs_updated_by_user_id
+num_associated_contacts
+num_associated_deals
+num_conversion_events
+num_conversion_events_cardinality_sum_d095f14b
```
Comment (jngiam) [inline hubspot/plugin.py:1489]: ditto

**Review by jngiam — APPROVED** [review]

File: hubspot/plugin.py [hunk 4]
```diff
@@ -353,9 +354,11 @@ class HubSpotPropertyValue:
     as_* methods can raise a ValueError if the value cannot be safely converted to the appropriate type.
 
     Empty strings are treated as 0 for int and float conversions.
+
+    Value can only be a string, int, float, or datetime.
```
Comment (vrv) [inline hubspot/plugin.py:358]:
```suggestion
    `value` property can only be a string, int, float, or datetime.
```

**Review by vrv — APPROVED** [review]

**Pull request merged**